webserver.md

```diff
@@ -0,0 +1,279 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+
+<div class="large-12 columns">
+# Build A Portable Web Server
+
+Most web applications are served from remote machines in data centers miles and miles away from the devices that are using them. Because we're used to high-speed internet connections, that distance can seem trivial and unnoticeable. 
+
+The Tessel 2 can run a server and deliver web applications to locally connected devices. It's quicker than those remote machines due to the immediate distance between the Tessel and devices connected to it. By the end of this article, we'll start a server on the Tessel and serve a web application that controls the Tessel LEDs through a local network.
```

Comment:
Is this really why we do this? Is it noticeable? I feel like we usually just do this because we can, or because it's more convenient to our mental model of interacting with a device through a page it hosts.